sources/packages/backend/libs/sims-db/src/entities/msfaa-number.model.ts

```diff
@@ -93,6 +93,7 @@ export class MSFAANumber extends RecordDataModel {
    * Student id related to this MSFAA.
    */
   @RelationId((msfaaNumber: MSFAANumber) => msfaaNumber.student)
+  @Column({ name: "student_id" })
```

Comment:
mm, why is this change? the name is joined in below code right, 
![image](https://github.com/bcgov/SIMS/assets/77353155/7c84fbe2-c405-4ae1-bdb9-3f119ab11310)